plugin/storage/integration/elasticsearch_test.go

```diff
@@ -259,8 +258,12 @@ func testElasticsearchStorage(t *testing.T, allTagsAsFields, archive bool) {
 	s.Fixtures = LoadAndParseQueryTestCases(t, "fixtures/queries_es.json")
 
 	if archive {
```

Comment:
* this flag should be deleted from tests
* TestElasticsearchStorage_Archive() should be deleted
* we only need to call IntegrationTestAll(), not manually call testArchiveTrace

Reply:
Done! PTAL